api/src/organizational-entities/domain/models/OrganizationForAdmin.js

```diff
@@ -89,6 +90,14 @@ class OrganizationForAdmin {
     this.code = code;
   }
 
+  get provinceCode() {
+    return this.#provinceCode;
+  }
+
+  set provinceCode(provinceCode) {
+    this.#provinceCode = provinceCode ? provinceCode.padStart(3, '0') : '';
+  }
```

Comment:
La suggestion de code ci-dessous :
* introduit des constantes pour éviter les « valeurs magiques »
* permet de respecter les valeurs `null` et `undefined` qui pourraient être passées en argument, sans cela elles seraient transformées en chaine vide

```suggestion
  const PAD_TARGET_LENGTH = 3;
  const PAD_STRING = '0';
  set provinceCode(provinceCode) {
    this.#provinceCode = provinceCode ? `${provinceCode}`.padStart(PAD_TARGET_LENGTH, PAD_STRING) : provinceCode;
  }
```

PS : Évidemment il faudrait mettre les constantes `PAD_TARGET_LENGTH` et `PAD_STRING` en début de fichier avec l'autre constante.

Reply:
Mais on veut pouvoir avoir des chaînes vide